# local_costmap 的 _xu 版本：含障碍物层 + 膨胀层，配合 costmap_common_params_xu.yaml 使用
local_costmap:
  global_frame: odom
  robot_base_frame: base_link

  update_frequency: 3.0
  publish_frequency: 1.0
  transform_tolerance: 0.5

  static_map: false
  rolling_window: true
  width: 3
  height: 3
  resolution: 0.05

  plugins:
    - {name: obstacle_layer, type: "costmap_2d::ObstacleLayer"}
    - {name: inflation_layer, type: "costmap_2d::InflationLayer"}
